src/coreclr/tools/Common/TypeSystem/Common/Properties/Resources.resx

```diff
@@ -195,4 +195,7 @@
   <data name="AmbiguousMatchUnsafeAccessor" xml:space="preserve">
     <value>Ambiguity in binding of UnsafeAccessorAttribute.</value>
   </data>
+  <data name="ExplicitMethodOverrideOnUnrelatedType" xml:space="preserve">
```

Comment:
Is this used somewhere?

Reply:
Not anymore, I removed the throw but I forgot to remove the resource, will fix in the next commit, thanks for reminding me!

Reply:
Should be gone now...